src/dos/dos_programs.cpp

```diff
@@ -327,7 +327,10 @@ class MOUNT : public Program {
 				int num = -1;
 				cmd->FindInt("-usecd",num,true);
 				int error = 0;
-				if (cmd->FindExist("-noioctl",false)) {
+				if (cmd->FindExist("-ioctl", false)) {
+					WriteOut(MSG_Get("MSCDEX_WARNING_IOCTL"));
+				}
+				if (cmd->FindExist("-noioctl", false)) {
 					MSCDEX_SetCDInterface(CDROM_USE_SDL, num);
 				}
```

Comment:
Previous code fell through to the else { } block if neither `-ioctl` or `-noioctl` were explicitly specified.  In the else { } block they got one of the `ioctl` MSCDEX interfaces. 

In this new code, there isn't a default, so they fall through to no MSCDEX interface. 
Looking ahead to SDL2, all of this is headed out the door anyway, so I realize I'm probably sweating too much on this; chances are this is all moot in a couple days anyway :-)